src/yuzu/main.cpp

```diff
@@ -2803,6 +2803,20 @@ void GMainWindow::ErrorDisplayDisplayError(QString error_code, QString error_tex
 }
 
 void GMainWindow::OnMenuReportCompatibility() {
+    const auto& caps = Common::GetCPUCaps();
+    const bool has_fma = caps.fma || caps.fma4;
+    const auto processor_count = std::thread::hardware_concurrency();
+    const bool has_4threads = processor_count == 0 || processor_count >= 4;
```

Comment:
```suggestion
    const bool has_4threads = processor_count >= 4;
```
It's highly unlikely that this will return 0

Reply:
As per cppreference, `This return 0 if the value is not well defined or not computable.`